packages/app/cypress/e2e/subscriptions/specChange-subscription.cy.ts

```diff
@@ -0,0 +1,357 @@
+import defaultMessages from '@packages/frontend-shared/src/locales/en-US.json'
+
+function getPathForPlatform (posixPath: string) {
+  if (Cypress.platform === 'win32') return posixPath.replaceAll('/', '\\')
+
+  return posixPath
+}
```

Comment:
This helper has been propagated a few times already, but you could import it from here instead: https://github.com/cypress-io/cypress/blob/10.0-release/packages/app/src/paths.ts